src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+//import java.awt.*;
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public String getColorSupplier() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
lets make Random as class level variable, not to create new instance of Random class every time we call getRandomColor() method